libs/langchain-community/src/stores/message/astradb.ts

```diff
@@ -0,0 +1,116 @@
+import { BaseListChatMessageHistory } from "@langchain/core/chat_history";
```

Comment:
Can you also add a separate entrypoint + mark it as optional to `langchain-community/langchain.config.js` and run `yarn build` in community?

https://github.com/langchain-ai/langchainjs/blob/main/CONTRIBUTING.md#adding-an-entrypoint